frame/im-online/src/lib.rs

```diff
@@ -131,7 +131,7 @@ pub mod ed25519 {
 
 /// The local storage database key under which the worker progress status
 /// is tracked.
-const DB_KEY: &[u8] = b"palette/im-online-worker-status";
+const DB_KEY: &[u8] = b"frame/im-online-worker-status";
```

Comment:
this might be a problem on a soft-upgrade ...

Reply:
We should probably remove this from the key all together right?